etcdserver/api/metrics/metrics.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The etcd Authors
```

Comment:
can this stay in etcdhttp but just export the handlers? having both health and metrics in a package called metrics is a little confusing (e.g., does Handler give me a metrics handler or health+metrics handlers)